--- augeas-1.0.0/lenses/shellvars.aug.orig	2013-01-07 19:43:51.945798396 +0100
+++ augeas-1.0.0/lenses/shellvars.aug	2013-01-07 20:06:02.779103989 +0100
@@ -188,6 +188,7 @@
       sc_incl "network/providers/*" .
       sc_excl "network-scripts" .
       sc_incl "network-scripts/ifcfg-*" .
+      sc_incl "network/interfaces/ifcfg-*" .
       sc_excl "rhn" .
       sc_incl "rhn/allowed-actions/*" .
       sc_excl "rhn/allowed-actions/script" .
